numpy/lib/histograms.py

```diff
@@ -238,6 +238,90 @@ def _get_outer_edges(a, range):
     return first_edge, last_edge
 
 
+def _get_bin_edges(a, bins, range, weights):
+    """
+    Computes the bins used internally by `histogram`.
+
+    Parameters
+    ==========
+    a : ndarray
+        Ravelled data array
```

Comment:
Someday we should settle on British or US English ;)

Reply:
Adding this British person to the team has likely not helped there